src/mminou.h

```diff
@@ -93,29 +102,277 @@ extern flag g_commandFileSilentFlag; /* For SUBMIT ... /SILENT */
 extern FILE *g_input_fp;  /*!< File pointers */
 extern vstring g_input_fn, g_output_fn;  /*!< File names */
 
-/*! print2 returns 0 if the user has quit the printout.
-  \warning:  never call print2 with string longer than PRINTBUFFERSIZE - 1  */
+/*!
+ * \fn flag print2(const char* fmt,...)
+ * \brief formatted output of a single line with optional page-wise display.
+ *
+ * It is important to understand that the submitted parameters result in a
+ * NUL-terminated string that contains at most one LF character, only allowed
+ * in final position.  A long line exceeding \ref g_screenWidth is broken down
+ * into multiple lines using a built-in line wrap algorithm.  But this must
+ * never be preempted by preparing parameters accordingly.
+ *
+ * The presence of the LF character controls whether a new line of output is
+ * generated, or the following output is padded right to the last line, except
+ * that output of more than \ref g_screenWidth characters is always closes the
+ * line.
+ *
+ * Although the output of a single line is the main goal of this function, it
+ * does a lot on the side, each effect individually enabled or disabled by
+ * various global variables that are honored, sometimes even updated.  We skim
+ * through these effects here in short:
+ *
+ * - (a) __Data embedding__.  The \p fmt parameter can contain simple text,
```

Comment:
```suggestion
 * - __Data embedding__.  The \p fmt parameter can contain simple text,
```
This is not an ordered list, and it has headers already, so I don't see any reason to attach numbers/letters to the elements of the list.

Reply:
Item (b) is referenced in step 2.  There was once some incentive to map each step to one of the goals shown in the first list, although this is carried out only to a very low extent now.
Can we somehow proceed without getting stuck on this imo irrelevant detail?  I know the description is imperfect, and formal details are the least of my concerns.  Or do I miss something important here?